src/docker-outside-of-docker/install.sh

```diff
@@ -273,6 +279,22 @@ if [ "${USE_MOBY}" = "true" ]; then
     fi
 fi
 
+
+docker_home="/usr/libexec/docker"
+cli_plugins_dir="${docker_home}/cli-plugins"
+
+install_compose_fallback(){
+    local url=$1
+    local compose_version=$2
+    local target_compose_arch=$3
+    local docker_compose_path=$4
```

Comment:
Do we need to explicitly pass these args?

Reply:
ya not reqd. , done